package.json

```
@@ -4,7 +4,7 @@
 	"description": "This is a Workshop about Api Testing in JavaScript",
 	"main": "index.js",
 	"scripts": {
-		"test": "mocha -t 5000"
+		"test": "mocha -t 5000 --reporter mochawesome --reporter-options reportDir=report,reportFilename=ApiTesting"
```

Please fix identation over all the file, for example:
```suggestion
	"test": "mocha -t 5000 --reporter mochawesome --reporter-options reportDir=report,reportFilename=ApiTesting"
```
When open a { use only one tab for identation